src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -419,4 +503,14 @@ public AttachToCaseEventData(
         merged.putAll(modifiedFields);
         return merged;
     }
+
+    private ServiceConfigItem getServiceConfig(CaseDetails caseDetails) {
```

Comment:
Why do we keep this validation if case type id has already been validated? It's better to include the result of that validation in `AttachToCaseEventData` and then just retrieve service config item based on it.